02_bubble/mohitnag/main_test.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBubble(t *testing.T) {
+	testData := []struct {
+		Scenario string
+		unSorted []int
+		expected []int
+	}{
+		{"Happy Day Scenario", []int{2, 5, 8, 1, 13, 1, 3}, []int{1, 1, 2, 3, 5, 8, 13}},
+		{"Slice containing negative numbers", []int{-1, 3, 5, -7}, []int{-7, -1, 3, 5}},
+		{"Empty Slice", []int{}, []int{}},
```

Comment:
Another possible edge case might be to have an input of `len(2)` because the inner loop starts from index 1 and through to len(s).

Reply:
Added test of slice of length 2